doc: add `scanblocks` to list of descriptor RPCs (#26352)
bitcoin/bitcoin: doc/descriptors.md

- `importdescriptors` takes as input descriptors to import into a descriptor wallet
  (since v0.21).
- `listdescriptors` outputs descriptors imported into a descriptor wallet (since v22).

> - `scanblocks` takes as input descriptors to scan for in blocks (since v25).

Reviewer: ```suggestion
- `scanblocks` takes as input descriptors and returns relevant blockhashes (since v25).
```
